### ISSUE TITLE
Tor Bridge Mode Support

### ISSUE THREAD

**Issue by thiras** [bisq-network/bisq#771]
Hello,

I live in a county which has highly censored internet connection. Direct Tor connection has been blocked. Therefor I have to use bridge mode. Unfortunately I couldn't find any settings on the launch screen about that.

It is a must have feature in my situation.

**Comment by ManfredKarrer** [comment]
We supported bridge mode a while ago but at least in China it did not help, so we removed it again. We will support soon plugable transports (and bridge then as well) so then it should be possible to use it in all circumstances.
For the moment you only might be able to get around it if you are a developer, knows about Tor (config) and run from source and change what needs to be changes. But I cannot give support here as I don't have the resources. Hope in about 3 months we have support for Tor everywhere.
